h2o-core/src/main/java/water/api/PutKeyServlet.java

```diff
@@ -33,7 +33,7 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
         return;
       }
 
-      InputStream is = ServletUtils.extractPartInputStream(request, response);
+      InputStream is = ServletUtils.extractInputStream(request, response);
```

Comment:
shouldn't it be read in try-catch with resources?

Reply:
good point, added